docs/pipelineruns.md

```diff
@@ -1386,6 +1386,7 @@ Your `PipelineRun`'s `status` field can contain the following fields:
     - [`kind`][kubernetes-overview] - Generally either `TaskRun` or `Run`.
     - [`apiVersion`][kubernetes-overview] - The API version for the underlying `TaskRun` or `Run`.
     - [`whenExpressions`](pipelines.md#guard-task-execution-using-when-expressions) - The list of when expressions guarding the execution of this task.
+  - `provenance` - A list of fields recording the key authenticated metadata about how a software artifact was built i.e. the source where remote resource came from. 
```

Comment:
Nit: provenance is not a list

How about something like  -  Metadata about resources used in the PipelineRun such as the source from where a remote pipeline definition was fetched

Reply:
SGTM. Updated. Thanks for the suggestion here. @dibyom

Reply:
Can we also update the descriptions for provenance in install.md and the feature flag comment itself to match?

Reply:
Sounds good. Updated! Thanks @dibyom !